vagrant/provisioning/roles/arkcase-app/templates/server.yml

```diff
@@ -139,4 +139,33 @@ payment:
     username: "{{ touchnet_username | default('') }}"
     password: "{{ touchnet_password | default('') }}"
     securelinkendpoint: "{{ touchnet_securelinkendpoint | default('') }}"
-    upaysiteid: "{{ touchnet_upaysiteid | default('') }}"
\ No newline at end of file
+    upaysiteid: "{{ touchnet_upaysiteid | default('') }}"
+
+################### Outlook email address for email ingestion - Mail Receiver ###################
+email:
+  should-delete-messages: false
+  should-mark-messages-as-read: false
+  max-messages-per-poll: 1
+  fixed-rate: 120000
+  #set some existing email here for testing
+  CASE_FILE.user: "{{ email_receiver_account_escaped | default('') }}"
+  CASE_FILE.password: "ENC({{ encrypted_email_receiver_password | default('') }})"
+  COMPLAINT.user: "{{ email_receiver_account_escaped | default('') }}"
+  COMPLAINT.password: "ENC({{ encrypted_email_receiver_password | default('') }})"
+  CONSULTATION.user: "{{ email_receiver_account_escaped | default('') }}"
+  CONSULTATION.password: "ENC({{ encrypted_email_receiver_password | default('') }})"
+  protocol: "imap"
+  fetch.folder: "Inbox"
+  host: "{{ email_host_imap }}"
+  port: {{ email_port_imap|int }}
+  debug: true
+  userId: "mail-service"
+  create:
+    case.enabled: {{ create_case_from_email_enabled | lower }}
+    complaint.enabled: {{ create_complaint_from_email_enabled | lower }}
+    consultation.enabled: false
+  enableBurstingAttachments: false
+  replyHandlers:
+    caseFileMailFilter: caseFileMailHandler
+    complaintMailFilter: complaintMailHandler
+    taskMailFilter: taskMailHandler
```

Comment:
Any property that isn't overridden (doesn't have a variable, should not be in the server file.